src/shared/components/plots/StackedBarPlot.tsx

```diff
@@ -29,11 +29,13 @@ export interface IStackedBarPlotProps {
     axisLabelX?: string;
     axisLabelY?: string;
     legendLocationWidthThreshold?: number;
+    percentage?:boolean;
+    stacked?:boolean;
```

Comment:
I think the component instead should be called `MultipleCategoryBarPlot` since it no longer is only stacked. what do you think? as it is now, the `stacked` prop confuses me, because I'm like "isnt the component stacked?"

Reply:
yeah, I'll rename it